packages/experiments/src/components/Shimmer/Shimmer.types.ts

```diff
@@ -139,11 +139,17 @@ export interface IShimmerStyles {
 export const enum ShimmerElementType {
   LINE = 'line',
   CIRCLE = 'circle',
-  GAP = 'gap'
+  GAP = 'gap',
+  line = 'line',
+  circle = 'circle',
+  gap = 'gap'
 }
 
 export const enum ShimmerElementVerticalAlign {
   CENTER = 'center',
```

Comment:
>CENTER [](start = 2, length = 6)

I am a bit confused. Why would we have two groups of enums ?

Reply:
The old ones are left until changes will be released and experiments package version will get bumped in odsp-common where ShimmerElementType.LINE is used in FileHoverCard. When that will be updated I will remove them from here.

Reply:
I thought about doing this, but I don't know if it makes sense as I know the only place outside the Fabric it has been used where it could break the contract with the component.
The component is still in experiments package and is if somebody put it into production code they were warned that it might brake due to API changes. So because I know were it is used in odsp-common I will just replace them in there and right after will remove them in here.

Reply:
Where would be the right place to put **@deprecated**?